docs/UserCommands.md

```diff
@@ -0,0 +1,44 @@
+## User Command Reference
+
+
+`@rocket user`
+
+## Options to specify input:
+
+* `-edit`
```

Comment:
A small usage thing: if we want to do `git`-style subcommands (which `argparse` does have full support for, I think) it would be more idiomatic to leave out the dash on the subcommands, so for instance we would want users to type `@rocket user edit --name Russell`.